docs/ci-integration/ci-integration.md

```diff
@@ -0,0 +1,118 @@
+
+`git` is used to help fetch remote targets, and also provides metadata for Earthly during your build.
+
+Because `earthly` will not install these for you, please ensure they are present before proceeding. These tools are very common, so many environments will already have them installed. If they are not, here are some installation instructions that may help:
+
+#### Linux
+
+To install `git`, you can typically use your distributions package manager. [This page](https://git-scm.com/download/linux) has installation instructions for most distributions.
+
+To install `docker`, use the most recent versions [directly from Docker](https://docs.docker.com/engine/install/#server). The versions packaged for many distributions tend to fall behind.
+
+#### macOS
+
+To install `git`, the easiest way is to install the "XCode Command Line Tools". If you open up `Terminal`, and type:
+
+```go
+git --version
+```
+
+Then macOS will prompt you to install these tools. You can also use the `git` provided installer or Homebrew, if you prefer. [Details can be found here](https://git-scm.com/download/mac).
+
+To install `docker`, [download and install Docker CE](https://hub.docker.com/editions/community/docker-ce-desktop-mac). Be sure to grab the correct installer depending on your CPU architecture.
+
+#### Windows
+
+To install `git`, use the  [MSI installer](https://gitforwindows.org/). This will provide `git`, and a Bash shell; which may prove more natural for using Earthly. You may also use your package manager of choice.
+
+To install `docker`, [download and install Docker CE](https://hub.docker.com/editions/community/docker-ce-desktop-windows). Both the HyperV and WSL2 backends are supported, but the WSL2 one is very likely to be faster.
+
+### Installation
+
+Once you have ensured that the dependencies are available, you'll need to install `earthly` itself.
+
+#### Bare Metal
+
+This is the simplest method for adding `earthly` to your CI. It will work best on dedicated computers, or in scripted/auto-provisioned build environments. You can follow our [regular installation guide](https://earthly.dev/get-earthly) to add `earthly` if desired.
+
+If you are going to mostly be working from a WSL2 prompt in Windows, you might want to consider following the Linux instructions for installation. This will help prevent any cross-subsystem file transfers and keep your builds fast. Note that the "original" WSL is unsupported.
+
+If your build host is persistent, it is recommended to install `earthly` as part of the new host's configuration, and not as part of your build. This will speed up your builds, since you do not need to download `earthly` each time; and it will also provide stability in case a future version of `earthly` changes the behavior of a command.
```

Comment:
I would add an example of what is a persistent build host vs what is not.

Reply:
I am not sure what kind of example you want here? Does this work?

> If your host is persistent *(a VM, physical box, or something that lives longer than a single build)*